packages/api/src/services/collateral-prices.ts

```diff
@@ -4,6 +4,8 @@ import { AppState, CurrencySymbol } from "..";
 type Config = {
   currency?: CurrencySymbol;
 };
+import { parseUnits } from "../libs/utils";
```

Comment:
Same comment as above about `@uma/common`'s `parseFixed` method.

Reply:
cool thanks, i just tried to do this, but im getting typescript errors. Apparently the version in common is not the same as what im using in this project, and there are changes to the api.  Going to leave it as is for now to avoid having to refactor a bunch of stuff related to this!